target/step159-2020-1/script.js

```diff
@@ -0,0 +1,4 @@
+function getClassID() {
+  var classId = Math.floor(Math.random() * 101);
```

Comment:
Pretty sure this will be changed later, just wanted to note that we should probably have a better hash to prevent duplicate class id's

Reply:
^ I feel like Java probably has a hashcode function or something?

Reply:
For sure will change later, was just to add the space but will look into hashcode :)

Reply:
Working on a solution for this but have to connect with Nina for this, will specify in the file itself that this is temporary.